tests/php/sync/test_class.jetpack-sync-full.php

```diff
@@ -781,8 +781,10 @@ function test_full_sync_can_sync_individual_comments() {
 		$comments = $synced_comments_event->args[0];
 
 		$this->assertEquals( 2, count( $comments ) );
-		$this->assertEquals( $sync_comment_id, $comments[0]->comment_ID );
-		$this->assertEquals( $sync_comment_id_2, $comments[1]->comment_ID );
+		$comment_IDs = array( $comments[0]->comment_ID, $comments[1]->comment_ID );
+
+		$this->assertTrue( in_array( $sync_comment_id, $comment_IDs ) );
```

Comment:
Should we use `$this->assertContains( $sync_comment_id, $comment_IDs );` here instead?